backend/metadata/databases/default/tables/public_Enrollment.yaml

```diff
@@ -14,9 +14,12 @@ object_relationships:
 - name: User
   using:
     foreign_key_constraint_on: UserId
-- name: motivationGradeByMotivationgrade
+- name: motivationRatingByMotivationrating
   using:
-    foreign_key_constraint_on: MotivationGrade
+    foreign_key_constraint_on: MotivationRating
```

Comment:
The default value of the column `MotivationRating` doesn't exists in the enum table `MotivationRating` which will result in the following error while creating a new record in the `Enrollment` table:
```
Error: Foreign key violation. insert or update on table "Enrollment" violates foreign key constraint "Enrollment_MotivationGrade_fkey"
```